images/cluster-api/Dockerfile

```diff
@@ -23,9 +23,16 @@ ENV IMAGE=${IMAGE_ARG}
 ENV ETCD_VER=v3.3.0
```

Comment:
I'm not sure we need to download & install etcd any longer. It would be useful to see if we can take it out and still have our tests pass. No need to do it as part of this PR, but could you file an issue to follow up on it?

Reply:
Opened https://github.com/kubernetes-sigs/cluster-api/issues/555 for that